File: decompiled/3.16.1/be/ehealth/businessconnector/therlink/domain/ProofTypeValues.java
package be.ehealth.businessconnector.therlink.domain;

public enum ProofTypeValues {
   EIDREADING("eidreading"),
   SISREADING("sisreading"),
   ISIREADING("isireading"),
   EIDSIGNING("eidsigning");

   private String value;

   private ProofTypeValues(String value) {
      this.value = value;
   }

   public String getValue() {
      return this.value;
   }
}

File: src/main/kotlin/org/taktik/connector/business/therlink/domain/ProofTypeValues.kt
package org.taktik.connector.business.therlink.domain

enum class ProofTypeValues private constructor(val value: String) {
    EIDREADING("eidreading"),
    SISREADING("sisreading"),
    ISIREADING("isireading"),
    EIDSIGNING("eidsigning"),
    EIDENCODING_HOUSECALL("eidencoding_housecall"),
    EIDENCODING_NOCARD("eidencoding_nocard"),
    EIDENCODING_TECHPROBLEM("eidencoding_techproblem"),
}
